token-service/src/main/java/org/p2p/token/service/repository/configurator/SetupTokenVisibilityConfigurator.kt

```diff
@@ -0,0 +1,15 @@
+package org.p2p.token.service.repository.configurator
+
+import org.p2p.core.token.MetadataExtension
+import org.p2p.core.token.Token
+import org.p2p.core.token.TokenExtensions
+
+class SetupTokenVisibilityConfigurator(
```

Comment:
I don't understand why we need this one liner classes

this is mapper work, no?